### PR TITLE
Initial support for Oculus SDK 1.3.0.0

### PR DESCRIPTION
**Pull request by crramirez**

### CODE REVIEW

File: pom.xml.releaseBackup [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (jherico) [inline pom.xml.releaseBackup]: This file should not be part of the source repository.  please update your .gitignore appropriately

File: release.properties [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#release configuration
```
Comment (jherico) [inline release.properties]: This file should not be part of the source repository.  please update your .gitignore appropriately

File: pom.xml [hunk 3]
```diff
@@ -1,66 +1,70 @@
...
+    <license>
+      <name>Oculus VR, Inc. Software Development Kit License Agreement</name>
+      <url>https://developer.oculusvr.com/license</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <scm>
+    <connection>scm:git:https://github.com/crramirez/jovr.git</connection>
+    <developerConnection>scm:git:git@github.com:crramirez/jovr.git</developerConnection>
```
Comment (jherico) [inline pom.xml]: I can't release to the maven central repository with this as the scm data.

File: src/main/java/com/oculusvr/capi/SessionStatus.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+import com.sun.jna.Memory;
+import com.sun.jna.Pointer;
+import com.sun.jna.Structure;
+
+/// Specifies status information for the current session.
+///
+/// \see ovr_GetSessionStatus
+///
+
+public class SessionStatus extends Structure implements Structure.ByValue {
```
Comment (jherico) [inline src/main/java/com/oculusvr/capi/SessionStatus.java]: this doesn't reflect the full structure of the SessionStatus type

```

File: src/main/java/com/oculusvr/capi/TrackerPose.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package com.oculusvr.capi;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.sun.jna.Pointer;
+import com.sun.jna.Structure;
+
+public class TrackerPose extends Structure implements Structure.ByValue {
+  public Posef Pose;
```
Comment (jherico) [inline src/main/java/com/oculusvr/capi/TrackerPose.java]: missing `unsigned int TrackerFlags;` and the struct padding.

File: src/main/java/com/oculusvr/capi/TrackingState.java [hunk 6]
```diff
@@ -13,6 +13,11 @@
   public PoseStatef[] HandPoses = new PoseStatef[2];
```
Comment (jherico) [inline src/main/java/com/oculusvr/capi/TrackingState.java]: this doesn't match the ovr struct in 1.3.... 

`CameraPose`, `LeveledCameraPose`, `RawSensorData`, `LastCameraFrameCounter` and `padding` all need to be removed
`StatusFlags` and `HandStatusFlags` are in the wrong location relative to the structure
`CalibratedOrigin` is missing

**Comment by jherico** [comment]
@crramirez This is a lot of good work, and it's almost there.  I do appreciate it very much.  

But there are some things that need to be fixed before I could merge and release this.  

On the plus side I was able to get my test code [here](https://github.com/OculusRiftInAction/jocular-examples) working with minimal work.

**Comment by crramirez** [comment]
@jherico I am happy I was able to help. At this time I am in a rush trying to update my app to 1.3 because was in 0.8 and it is incompatible with the actual runtime.

My first goal was precisely make your demo run. I uploaded those changes as well in my fork but, late at night I discovered that my branch was not from your latest commit, and I confused trying to make a pull request. I am very new in Git and I am afraid to ruin something.

My application uses JOGL and for me it is important to have your library up to date, not only because my app need it but also it is based in an open source project called Open Wonderland which I will submit the changes when I finish with the 1.3 migration. This includes your library.

I made a first attempt pull request because I didn't know if you are already working on the update, but I couldn't wait. Also due new restrictions in GitHub I cannot send you a message directly to ask you about it.

Today I will focus in making my application run and publish it. 

Then I will work in polish my pull request, and you helped me with your corrections.

There are changes that are not intended to be in your repo, but a don't know how submit only a few in the pull request. For example in the pom I changed scm to be able to release in my fork. Also I had to disable the gpg signing process because I couldn't make it run in Windows.

Thank you for all your comments I will be working to polish the library changes and also to do best practices, I know that in a rush the code is not as elegant it should be.

Regards,
Carlos

File: pom.xml (hunk 3 above)
Comment (crramirez) [inline pom.xml]: Yes, exactly as you said.

**Comment by jherico** [comment]
Thanks, I've actually thought about it and I can merge the PR as it stands now and then make some local changes to fix some of the issues and add some more polish.    Thanks again for the work.

**Comment by crramirez** [comment]
Excellent!!!

**Pull request merged**